components/Resource.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import s from 'styled-components'
+
+import { Title, StyledLink, ContentIndent } from './shared'
+import { LIBRE_BOLD } from '../utils/font'
+
+const ResourceWrapper = s.div`
+  margin-top: 1rem;
+  background: #F5F5F5;
+  padding: 5rem 2rem;
+  margin-bottom: 2rem;
+`
+const ResourceTitle = s.h5`
+  ${LIBRE_BOLD}
+  color: #464242;
+
+  transition: 0.3s;
+
+  :hover {
+    color: #B5B4B4;
+    transition: 0.3s;
+  }
+`
+
+const NumberCircle = s.div`
```

Comment:
make this component mobile-friendly, if u shrink ur browser size, u will see that the circle covers the text, so reduce `width` and `height` to let's say `1rem` if the screen size is small (<768px). For how to do it, https://github.com/dailypenn/projects.thedp.com/blob/master/src/components/shared/Flex.js#L9, or google media query css